.github/benchmark-configs.json

```diff
@@ -167,5 +167,59 @@
       "data_instance_config": "4vCPU, 32G Mem, 16G Heap"
     },
     "baseline_cluster_config": "x64-r5.xlarge-single-node-1-shard-0-replica-baseline"
+  },
+  "id_11": {
+    "description": "Search only test-procedure for big5 with concurrent segment search setting enabled",
+    "supported_major_versions": ["2", "3"],
```

Comment:
i think this will only be 3 since snapshot used is created on 3.x version

Reply:
@sohami And do we want to set slice count to something or leave it default?

Reply:
we can leave it as is for now since we want to compare if there is any overhead of the default decider.